helpers/validateTokenExpiry.ts

```diff
@@ -0,0 +1,13 @@
+import jwt, { TokenExpiredError } from "jsonwebtoken";
+
+export const validateTokenExpiry = (token: string) => {

```

Comment:
Możemy tutaj dodać dodatkową walidację przed try catchem

 if (!process.env.TOKEN_SECRET || !token) {
    throw new Error("TOKEN_SECRET or token is not provided");
  }